core/server/src/test/java/alluxio/master/file/PermissionCheckerTest.java

```diff
@@ -194,6 +194,14 @@ public void before() throws Exception {
     mPermissionChecker = new PermissionChecker(sTree);
   }
 
+  // Helper function to create a path and set the permission to what specified in option.
+  private static void createAndSetPermission(String path, CreateFileOptions option)
+      throws Exception {
+    InodeTree.CreatePathResult result = sTree.createPath(new AlluxioURI(path), option);
+    result.getCreated().get(result.getCreated().size() - 1)
```

Comment:
because `sTree.createPath()` applies umask when creating new inodes. Some of the permission bits may be wiped-off after `createPath` even they are set in the options.